ToolLibrary.Standart/DoubleLinkedList.cs

```diff
@@ -0,0 +1,330 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace ToolLibrary.Standart
+{
+    /// <summary>
+    /// Represents a node in a <see cref="DoubleLinkedList{T}"/>.
+    /// </summary>
+    /// <typeparam name="T">Specifies the value type.</typeparam>
+    public class DoubleLinkedListNode<T>
+    {
+        internal DoubleLinkedListNode(T value)
```

Comment:
why didn't you public this constructor?

Reply:
Createing a new node outside of list and pass into list for removing will cause changes a Count, though list wasn't changed. It could be handled using some kind of belongs attribute or trying to find item within list).  It should be part of this functionality.